artemis/src/main/java/tech/pegasys/artemis/DepositCommand.java

```diff
@@ -81,44 +99,144 @@ public void generate(
           int validatorCount,
       @Option(
               names = {"--output-file", "-o"},
+              paramLabel = "<FILE>",
               description =
-                  "File to write validator keys to. Keys are printed to std out if not specified")
-          String outputFile) {
+                  "File to write unencrypted validator keys to. Keys are printed to std out if not specified")
+          String outputFile,
+      @Option(
+              names = {"--encrypt-keys", "-e"},
+              defaultValue = "false",
```

Comment:
False doesn't seem like the right default here.  Writing unencrypted keys to disk is a really bad idea.

Reply:
`--encrypt-keys` defaults to true now.